### ISSUE TITLE
mozim crate fails to build on i686

### ISSUE THREAD

**Issue by Luap99** [nispor/mozim#11]
From the netavark build log:
```
Compiling mozim v0.1.0
error[E0277]: the trait bound `i32: From<u32>` is not satisfied
   --> /builddir/build/BUILD/netavark-1.5.0/vendor/mozim/src/socket.rs:289:17
    |
289 |         tv_sec: timeout.into(),
    |                 ^^^^^^^ ---- required by a bound introduced by this call
    |                 |
    |                 the trait `From<u32>` is not implemented for `i32`
    |
    = help: the following other types implement trait `From<T>`:
              <i32 as From<NonZeroI32>>
              <i32 as From<bool>>
              <i32 as From<i16>>
              <i32 as From<i8>>
              <i32 as From<nix::time::ClockId>>
              <i32 as From<nix::unistd::Pid>>
              <i32 as From<socket2::Domain>>
              <i32 as From<socket2::Protocol>>
            and 5 others
    = note: required for `u32` to implement `Into<i32>`
For more information about this error, try `rustc --explain E0277`.
error: could not compile `mozim` due to previous error
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:50: build] Error 101
error: Bad exit status from /var/tmp/rpm-tmp.sLYjfo (%build)
RPM build errors:
Bad exit status from /var/tmp/r
```
original report in netavark: https://github.com/containers/netavark/issues/578

**Comment by cathay4t** [comment]
Issue fixed in PR #14 and crate release 0.2.1.

**Comment by Luap99** [comment]
thanks
